src/components/CheckoutCard.vue

```diff
@@ -33,6 +34,9 @@ export default class CheckoutCard<
     protected statusScreenMainAction: () => void = () => console.warn('statusScreenMainAction not set.');
 
     protected async created() {
+        this.hasCurrencyInfo = this.request.paymentOptions.length > 1
+            && (!history.state || !history.state[HISTORY_KEY_SELECTED_CURRENCY]);
```

Comment:
I don't quite understand the naming of this variable? Why is `hasCurrencyInfo` only true when there are _more_ than one paymentOption? Doesn't the single payment option also include a currency info?

Reply:
Yes, exactly. This has always been the case.